### ISSUE TITLE
`document.write()` usage not caught

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-linter#4148]
In https://github.com/mozilla/addons-linter/issues/4142, we removed the `NO_DOCUMENT_WRITE` error (and the ESLint rule) because it was already covered by https://github.com/mozilla/eslint-plugin-no-unsanitized (`UNSAFE_VAR_ASSIGNMENT`). It looks like it isn't working as expected, though.

## STR

Write an add-on with a JS file that contains:

```js
document.write("foo");
```

## Expected

A warning about `document.write()` should be emitted.

## Actual

No error when validating the add-on with the linter v4.7.0 (latest version currently):

```
$ ./bin/addons-linter test
Validation Summary:

errors          0
notices         0
warnings        0
```

This looks like a regression because there is an error when validating the add-on with the linter v4.6.0:

```
$ ./bin/addons-linter test
Validation Summary:

errors          0
notices         0
warnings        1

WARNINGS:

Code                Message        File       Line   Column
NO_DOCUMENT_WRITE   Use of         index.js   1      1
                    document.wr…
                    ite strongly
                    discouraged.
```

**Comment by Rob--W** [comment]
`document.write("fixed string")` is not that big of a deal.

Have you tried a variable? When I reviewed the patch I saw that there is already a test case that confirms that a message is emitted for dynamic document.write values.

document.write containing `<` may be a bigger deal, but the same can be said for innerHTML and arbitrary method invocations on `document`.

**Comment by ioanarusiczki** [comment]
@willdurand I checked this morning - linter version is still 4.7.0 and if I use document.write("Hello World"); this gets undetected. 

With a variable I get a linter warning "UNSAFE_VAR_ASSIGNMENT"
var mytext = "Hello again";
document.write(mytext);

**Comment by willdurand** [comment]
I haven't released a new version of the linter yet, fwiw

**Comment by ioanarusiczki** [comment]
@willdurand Yeah, pls let me know and I'll re-check.

**Comment by willdurand** [comment]
@ioanarusiczki `4.8.0` is on its way

**Comment by ioanarusiczki** [comment]
@willdurand This is fixed on 4.8.0 

![docwrite](https://user-images.githubusercontent.com/33448286/151794365-2383cf47-01b9-4eec-8b39-7f859d8610fc.png)
